src/toggle-attribute/README.md

```diff
@@ -64,17 +64,44 @@ import '@travelopia/web-components/dist/toggle-attribute/style.css';
 		<option value="">Select</option>
 		<option value="First">First</option>
 		<option value="Second">Second</option>
+		<option value="Third">Third</option>
+		<option value="Fourth">Fourth</option>
+		<option value="Fifth">Fifth</option>
 		<option value="All">All</option>
 	</select>
 </tp-toggle-attribute>
 
 <div class="toggle-target-variable" data-toggle-value="First,All">
 	Toggled First
 </div>
+<div class="toggle-target-variable" data-toggle-value-non-empty>
+	<!-- add the data-toggle-value-non-empty attribute to toggle on with all non empty values. -->
+	This will be toggled for all non empty values.
+</div>
 <div class="toggle-target-variable" data-toggle-value="Second,All">
 	Toggled Second
 </div>
 
+<!-- Toggle for non empty values. -->
+<p>Toggle with non empty values</p>
+
+<!-- Select value -->
+<tp-toggle-attribute target=".toggle-target-non-empty" value-non-empty>
```

Comment:
@abhishekxix Lets rename to `non-empty-value="yes"` to match our other components

Reply:
@abhishekxix Can you name the attributes: `non-empty-value` and `data-non-empty-value`?

Reply:
Fixed in [e5827f1](https://github.com/Travelopia/web-components/pull/79/commits/e5827f189f53ac713fd2ebccc9c06bc34564a4c8)